.github/workflows/release-step-3.yml

```diff
@@ -150,18 +150,14 @@ jobs:
 
   publish-aws-lambda:
     name: "Publish AWS Lambda"
+    permissions:
```

Comment:
You mentioned that renaming this file would break auth. Maybe add a comment at the very top that this file should not be renamed (or auth needs to be updated in that case)?

Reply:
https://github.com/elastic/apm-agent-java/pull/3696/commits/ee88f356a141ecd3a2d1e46ec9b38d5d62669409